src/views/admin/RolleManagementView.vue

```diff
@@ -0,0 +1,95 @@
+<script setup lang="ts">
+  import {
+    RolleResponseMerkmaleEnum,
+    useRolleStore,
+    type RolleResponse,
+    type RolleStore
+  } from '@/stores/RolleStore'
+  import { computed, onMounted, type ComputedRef } from 'vue'
+  import ResultTable from '@/components/admin/ResultTable.vue'
+  import { type Composer, useI18n } from 'vue-i18n'
+  import type { VDataTableServer } from 'vuetify/lib/components/index.mjs'
+  import {
+    useOrganisationStore,
+    type Organisation,
+    type OrganisationStore
+  } from '@/stores/OrganisationStore'
+
+  const rolleStore: RolleStore = useRolleStore()
+  const organisationStore: OrganisationStore = useOrganisationStore()
+
+  const { t }: Composer = useI18n({ useScope: 'global' })
+
+  type ReadonlyHeaders = InstanceType<typeof VDataTableServer>['headers']
+  const headers: ReadonlyHeaders = [
+    { title: t('admin.rolle.rollenname'), key: 'name', align: 'start' },
+    { title: t('admin.rolle.rollenart'), key: 'rollenart', align: 'start' },
+    { title: t('admin.rolle.merkmale'), key: 'merkmale', align: 'start' },
+    {
+      title: t('admin.schulstrukturknoten.schulstrukturknoten'),
+      key: 'administeredBySchulstrukturknoten',
+      align: 'start'
+    }
+  ]
+
+  const transformedRollenAndMerkmale: ComputedRef<
+    {
+      rollenart: string
+      merkmale: string
+      id: string
+      createdAt: string
+      updatedAt: string
+      name: string
+      administeredBySchulstrukturknoten: string
+    }[]
+  > = computed(() => {
```

Comment:
```suggestion
  const transformedRollenAndMerkmale: ComputedRef<RolleResponse[]> = computed(() => {
```
Spricht was dagegen?

Reply:
Ja, rollenart und merkmale sind keine Strings normalerweise... die sind Enums in der Response und werden in dieser Methode zu einem String